test/release/examples/primers/modules.chpl

```diff
@@ -778,19 +817,46 @@ module UsesTheUser2 {
   writeln("Start of UsesTheUser2's module-level code");
   func2();
   {
-    use UsesTheImporter;
+    use UsesTheUser3;
   }
   writeln("End of UsesTheUser2's module-level code");
 }
 
+/* In addition, ``public use`` statements `re-export` the symbols in the used
+   module.  This means that the used symbols can also be treated as though they
+   are defined in the scope with the ``use`` for the purpose of accesses from
+   outside that module.  For example, here it will also appear as though the
+   module ``ModuleThatIsUsed`` is a submodule of ``UserModule2``, so other
+   scopes can also access it in that manner.
+*/
+module UsesTheUser3 {
+  proc func3() {
+    use UserModule2;
+    UserModule2.ModuleThatIsUsed.publiclyAvailableProc();
+    // The above is available due to the ``public use`` of ``ModuleThatIsUsed``
+    // in ``UserModule2``.
+  }
+
+  // These lines are to ensure everything gets tested regularly
+  writeln("Start of UsesTheUser3's module-level code");
+  func3();
+  {
+    use UsesTheImporter;
+  }
+  writeln("End of UsesTheUser3's module-level code");
+
+}
+
 /* Import statements can also be labeled as being either ``public`` or
    ``private``.  By default ``import`` statements are also ``private``.
    However, ``public import`` statements have a different meaning than ``public
-   use`` statements do - ``public import`` statements `re-export` the symbols in
-   the imported module.
+   use`` statements do - while ``public import`` statements still `re-export`
```

Comment:
Did you mean to use two backtics in "re-export" ?

Reply:
Nah, that was intentionally to make it italicized versus turned into a code snippet